docs/docker.md

```diff
@@ -39,5 +35,14 @@ Besides, currently `cvdr` supports using SOCKS5 proxy with the flag like
 
 ## Use cloud orchestrator by cvdr
 
-Please follow [cvdr.md](cvdr.md). The URL of running cloud orchestrator should
-be `http://${CLOUD_ORCHESTRATOR_IP_ADDRESS}:8080`.
+Please use the configuration file `scripts/docker/cvdr.toml` and follow the
+steps at [cvdr.md](cvdr.md). The URL of running cloud orchestrator should
+be `http://${CLOUD_ORCHESTRATOR_IP_ADDRESS}:8080`. For example:
+```bash
+CVDR_USER_CONFIG_PATH=./scripts/docker/cvdr.toml \
+./cvdr \
+--service_url=http://${CLOUD_ORCHESTRATOR_IP_ADDRESS}:8080 \
```

Comment:
if using a config file `scripts/docker/cvdr.toml` already then we can skip declaring the `service_url`, `proxy` and `zone` flags on every execution, make these values part of the configuration file as well.

Reply:
Done, thanks!